### PR TITLE
feat(secnav): design review for secondary navigation component

### PR DESCRIPTION
**Pull request by 100stacks**
## Description

Design Review for `Secondary Navigation` component.

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2092

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: docs/components/secnav/test.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <div id="vue-secNavDemo">
```
Comment (100stacks) [inline docs/components/secnav/test.html:7]: @danielmdesigns, please see the deploy preview full-page demo:

https://deploy-preview-785--helix-ui.netlify.app/components/secnav/test.html

**Review by BY00565233 — APPROVED** [review]
Dev LGTM

File: docs/components/secnav/test.html (hunk 1 above)
Comment (danielmdesigns) [reply docs/components/secnav/test.html:7]: secondary-nav-padding

Looks good except the top/bottom padding is 4px and should be 8px. Screenshot attached.

**Review by 100stacks — COMMENTED** [review]
### @danielmdesigns, please see the updates for the Secondary Navigation component based on your feedback.

#### https://deploy-preview-785--helix-ui.netlify.app/components/secnav/test.html

**Comment by danielmdesigns** [comment]
@100stacks i'm going to go ahead and approve these knowing that we have some updated changes coming down the pipe. let's go ahead and get these merged and we'll finalize once we have fabric updates available

**Review by danielmdesigns — APPROVED** [review]
LGTM! Will provide more feedback w/ updates once new Fabric changes are available

**Pull request merged**